.github/workflows/govulncheck.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI vulnerability detection with govulncheck
+
+on:
+  pull_request:
+    branches:
+      - main
+      - "release-4.1[9]"
```

Comment:
We don't have 4.19 branch yet on the repo. Shouldn't this be `release-4.1[8]`?

Reply:
Right. Thanks